crates/fuel-core/src/service/adapters/txpool.rs

```diff
@@ -91,6 +92,21 @@ impl fuel_core_txpool::ports::PeerToPeer for P2PAdapter {
         }
     }
 
+    fn new_tx_subscription(&self) -> BoxStream<Vec<u8>> {
+        use tokio_stream::{
+            wrappers::BroadcastStream,
+            StreamExt,
+        };
+        if let Some(service) = &self.service {
+            Box::pin(
+                BroadcastStream::new(service.subscribe_new_tx_subscription())
+                    .filter_map(|result| result.ok()),
+            )
+        } else {
+            fuel_core_services::stream::IntoBoxStream::into_boxed(tokio_stream::pending())
```

Comment:
can this be `fuel_core_services::stream::pending()` ?

Reply:
Done 👍